### PR TITLE
Add 'donate' badge to README and improve wording

### PR DESCRIPTION
**Pull request by astrofrog**
Paging @kelle @eteq @taldcroft - this link to the NumFOCUS donation page for astropy.

This is what it looks like: https://github.com/astrofrog/astropy/blob/donate/README.rst

### CODE REVIEW

**Comment by kelle** [comment]
I had to read the readme 4 times before I saw the badge.  I think it should go down in the Project Status section, next to "powered by Numfocus" and some text should be added. Here's some text inspired by the text on the NumFocus donate page:

> The Astropy Project is sponsored by NumFOCUS, a 501(c)(3) nonprofit in the United States. Your tax-deductible donation to the Astropy Project will support our mission to promote sustainable high-level code base for the astronomy community, open code development, educational materials, and reproducible scientific research.

File: README.rst [hunk 1]
```diff
@@ -33,14 +35,14 @@ Project Status
...
 .. image:: https://ci.appveyor.com/api/projects/status/ym7lxajcs5qwm31e/branch/master?svg=true
     :target: https://ci.appveyor.com/project/Astropy/astropy/branch/master
     :alt: Astropy's Appveyor Status
 
+.. image:: https://coveralls.io/repos/astropy/astropy/badge.svg
+    :target: https://coveralls.io/r/astropy/astropy
+    :alt: Astropy's Coveralls Status
+
 For an overview of the testing and build status of all packages associated
 with the Astropy Project, see http://dashboard.astropy.org.
```
Comment (kelle) [inline README.rst]: I had to read the readme 4 times before I saw the badge. I think it should go down in the Project Status section, next to "powered by Numfocus" and some text should be added. Here's some text inspired by the text on the NumFocus donate page:

    The Astropy Project is sponsored by NumFOCUS, a 501(c)(3) nonprofit in the United States. Your tax-deductible donation to the Astropy Project will support our mission to promote sustainable high-level code base for the astronomy community, open code development, educational materials, and reproducible scientific research.

Comment (eteq) [inline README.rst]: 👍

**Comment by astrofrog** [comment]
@eteq @kelle @taldcroft - I've revived this PR and have tried to make the information more prominent. I've taken the opportunity to also clean up the README file in other ways. You can preview the current version here:

https://github.com/astropy/astropy/blob/cd015196b785952a5cbc585c50f42cfc3caa9ea4/README.rst

Does this seem better now?

**Comment by pllim** [comment]
Can we move that section even more up, above contributing? 😬 🤑

**Comment by eteq** [comment]
Personally I think it's better where it is, because I think we want to (and do!) value contributing more than 💸 (if only just!), because that's then work we don't have to "manage".

To get this in for the next release I'm going to merge it.  I think the two requested changes were addressed, so will dismiss them to merge.  But if @kelle @taldcroft or @pllim want to make it more prominent I suggest a follow-on PR.

**Pull request merged**